java-containeranalysis/.OwlBot-hermetic.yaml

```diff
@@ -18,11 +18,11 @@ deep-remove-regex:
 - "/java-containeranalysis/google-cloud-containeranalysis/src/.*/java/com/google/cloud/devtools/containeranalysis/v1"
 - "/java-containeranalysis/samples/snippets/generated"
 - "/java-containeranalysis/proto-google-cloud-containeranalysis-v1/src"
+- "/java-containeranalysis/proto-google-cloud-containeranalysis-v1beta1/src"
 # TODO: remove resource name classes in next major update
 
 deep-preserve-regex:
 - "/java-containeranalysis/google-.*/src/test/java/com/google/cloud/.*/v.*/it/IT.*Test.java"
-- "/java-containeranalysis/google-cloud-containeranalysis/src/.*/java/com/google/cloud/devtools/containeranalysis/v1beta1"
 - "/java-containeranalysis/proto-google-cloud-containeranalysis-v.*/src/main/java/com/google/containeranalysis/v.*/.*Name.*.java"
```

Comment:
The resource names are also generated based on the proto files. Do you think we can also remove these from being preserved and update them to the latest as part of the `feat!` PR?

Reply:
Yes, removed the non-existent resouce names.